nestor_api/lib/config.py

```diff
@@ -63,13 +61,23 @@ def _resolve_variable(template: str, variables: dict, path: str) -> str:
 
     # Resolve pattern '{{variable}}'
     referenced_names = re.findall(r"{{([\w\-.]+)}}", template)
-    for var_name in referenced_names:
-        var_value = variables.get(var_name)
-        if var_value is not None:
-            if not isinstance(var_value, str):
-                raise ConfigurationError(path, "Referenced variable should resolved to a string")
-
-            final_value = re.sub(f"{{{{{var_name}}}}}", var_value, final_value)
+    if len(referenced_names) > 0:
```

Comment:
Not sure that we need this extra check as we won't go into the loop below if length equals 0